### ISSUE TITLE
Updating paket.exe in magic mode fails with access denied

### ISSUE THREAD

**Issue by atlemann** [fsprojects/Paket#3202]
### Description

We're stuck on version v5.163.0 which breaks our build and are trying to update paket to v5.163.1 by running.

```bash
.paket/paket.exe --self --run --version

Checking Paket version (downloading latest stable)...
Trying self update
Starting download of bootstrapper from https://github.com/fsprojects/Paket/releases/download/5.163.1/paket.bootstrapper.exe
Self update failed. Resetting bootstrapper.
Access to the path '.paket\paket.exe' is denied. (Github - cached)
Paket version 5.163.0
```

This is on Windows. Haven't tried on linux.

### Repro steps

Run this Paket command in magic mode:  `.paket/paket.exe --self --run --version`

### Expected behavior

The newest version to be downloaded.

### Actual behavior

Access denied on `.paket/paket.exe`

### Known workarounds

None

**Comment by forki** [comment]
/cc @vbfox

Atle Rudshaug <notifications@github.com> schrieb am Mo., 14. Mai 2018,
12:48:

**Comment by atlemann** [comment]
It would help if `.paket/paket.exe --clear-cache` would delete the temporary `paket.exe` files as a workaround. Now I'll have to go into all the build agents and remove `%temp%/paket....` manually unless there's something i'm missing.

**Comment by forki** [comment]
There is this bootstrapper parameter that I sent you on Twitter. Look into
docs for details. And sorry for breaking you

Atle Rudshaug <notifications@github.com> schrieb am Mo., 14. Mai 2018,
12:58:

**Comment by atlemann** [comment]
Yes, but we're in magic mode and then `--max-file-age` is unknown parameter:

```bash
$ .paket/paket.exe --max-file-age=0
ERROR: unrecognized argument: '--max-file-age=0'.
```

**Comment by atlemann** [comment]
If renaming `paket.exe` back to `paket.bootstrapper.exe` the parameter works and we would get the full size paket.exe in the `.paket` folder. Then what should I do with that?

No prob breaking the build, it was just a bit confusing that the build suddenly stopped working.

**Comment by forki** [comment]
yeah are having trouble with the targets file -  why is your build running install?
restore should not touch that file!?

**Comment by atlemann** [comment]
`paket.exe restore` changes the Paket.Restore.targets file

**Comment by dsyme** [comment]
We will move to paket distribute only as a .NET tool
